### ISSUE TITLE
Why we need to ensure the data format is adjusted with 10,000 for normalization correctness?

### ISSUE THREAD

**Issue by HelloWorldLTY** [Teichlab/celltypist#138]
Hi, thanks for your great work! I notice that celltypist requires our input data to be normalized with 10,000 per cell counts, but it seems that when computing the loss, celltypist does not have any assumption of input data, is it only a result from empricial experiments? Can I directly input pca for prediction?

Moreover, starting from line 309 in celltypist/classifier.py, it seems that we only consider adata[0:1000]. How do you determine 1000? Thanks a lot.

**Comment by ChuanXu1** [comment]
@HelloWorldLTY, CellTypist records the scaling parameters (mean and sd) for each gene in the training dataset, and then re-applies them to the query data. 10,000 is an arbitrary choice, but CellTypist has to make sure these parameters which are derived from 10,000-based data will be applied to the same 10,000-based data.

For the second question, it's just a quick way to check whether the data contains negative values.
